src/csm_testing/tests/iuf_stages_cleanup/__init__.py

```diff
@@ -0,0 +1 @@
+from .__main__ import get_nexus_credentials, get_ca_certificates
```

Comment:
I don't see any other code trying to import from this module. Why is this line needed?